app/views/layouts/_flash_messages.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="usa-alert usa-alert--success usa-alert--slim">
     <div class="usa-alert__body">
       <p class="usa-alert__text">
-        <%= flash[:notice] %>
+        <%= sanitize flash[:notice] %>
```

Comment:
[praise] excellent job finding a safe way to implement this. I forgot about `sanitize`